CHANGELOG.md

```diff
@@ -4,6 +4,28 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.0.0-rc4 (2020-12-02)
+
+This release candidate contains bugfixes since the previous release
+candidate, as well as additional minor features. It adds checks of
```

Comment:
unclear what "checks of configuration" is or refers to. Maybe "It improves configuration validation for background jobs, ..."

Reply:
Reformulated.